### DISCUSSION TITLE
Not able to use OTA updates, at all?

### DISCUSSION THREAD

**Discussion by hyperion007** [Koenkk/zigbee2mqtt#19380] [General]
Hi,
I know this is a topic that has been beaten to death but I really want to understand why I can't get it to work.
It worked in the beginning, several years ago, when I first started using Z2MQTT and Node-RED.

I have searched and read everything I could find on this topic but still can't get it to work.

First of all, some of my devices have at some point, a long time ago, been "flagged" as having a firmware update available. How can I reset this status? Because it says either "Device did not respond to OTA request" or "no image available" 

And I have set ikea_ota_use_test_url to both true and false but it doesn't matter.
I have also tried different OTA index override file name but it didn't work.

And now I get this error when I press "check for new updates" on any device:  
Failed to check if update available for 'AcousticPanelRear' (ENOENT: no such file or directory, open '/opt/zigbee2mqtt/data/ ')
I think it's the "space" after .../data/ that is the issue but I can't seem to get rid of it. 

I tried to enter "https://github.com/Koenkk/zigbee-OTA/blob/master/index.json" in the "OTA index override file name" field.
but that only results in this error when I press "check for new updates":
Failed to check if update available for 'AcousticPanelRear' (localIndex.concat(...).map is not a function)

Finally I have created a copy of my_index.json and put it in /opt/zigbee2mqtt/data/ and put my_index.json in the "OTA index override file name" field.

This at least doesn't throw errors but no updates are available or devices doesn't respond to OTA request. Some devices have a newer FW than what is available...

Any suggestions here? 

Thanks.
Zigbee2MQTT version
1.32.2 commit: 1ec1e57
Coordinator type
zStack3x0
Coordinator revision
20210120
Frontend version
0.6.133

**Comment by el-bakkali** [comment]
did you ever get this resolved ?
im having same issue

**Reply by hyperion007** [reply]
Yes it started to work again after some update. Works for IKEA stuff at least and that's mostly what I have. would be nice to update GLEDOPTO stuff too but hey, they work as is so I don't know if an update would improve anything :)
